packages/console/src/components/PlanUsage/index.tsx

```diff
@@ -26,7 +25,7 @@ import {
 } from './utils';
 
 type Props = {
-  readonly periodicUsage?: NewSubscriptionPeriodicUsage;
+  readonly periodicUsage: NewSubscriptionPeriodicUsage | undefined;
```

Comment:
Is this change necessary?

Reply:
Yes, it indicates that `periodicUsage` property is required when using this component, but the value can be undefined.